x-pack/plugins/cases/server/services/cases/transform.ts

```diff
@@ -178,10 +183,13 @@ export function transformSavedObjectToExternalModel(
     caseSavedObject.references
   );
 
+  const severity = SEVERITY_ESMODEL_TO_EXTERNAL[caseSavedObject.attributes?.severity];
```

Comment:
I think we should default it `SEVERITY_ESMODEL_TO_EXTERNAL[caseSavedObject.attributes?.severity] ?? CaseSeverity.LOW`

Reply:
iirc I ran into some problems defaulting, I have to test it again.

Reply:
Ok, just needed to update a test that missed this default value. All good 👍